pmoves/tests/test_common/test_bootstrap.py

```
@@ -0,0 +1,65 @@
+        service_dir = str(Path(fake_file).resolve().parent)
+        assert service_dir in sys.path
+        # The services dir (parent[1]) should also be in sys.path
+        services_dir = str(Path(fake_file).resolve().parents[1])
+        assert services_dir in sys.path
+    finally:
+        sys.path[:] = original_path
+
+
+def test_bootstrap_with_extra_roots():
+    """bootstrap_import_paths adds extra_roots to sys.path."""
+    extra = Path("/tmp/fake_extra_root")
```

Ruff (0.15.10)

[error] Probable insecure usage of temporary file or directory: "/tmp/fake_extra_root"

(S108)